src/Apps/Sell/Routes/NewRoute/index.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from "react"
```

Comment:
Can we name the folder `Routes/New` vs doubling up?

Reply:
Yep, definitely. Can de-duplicate the naming here.

Reply:
I tend to always consider components used in sub routes to be placed in a `Foo/Routes/Route/Components` folder, treating each sub route as its own little sub-app. But this division isn't strict, especially if one is certain that the sub route will never scale out to any largish size. But that said, we should follow patterns as seen elsewhere (just in case), so yeah. 

TLDR: use yr best judgement.

Reply:
All of the routes are single files now. 👍🏼